packages/ipfs/.aegir.js

```diff
@@ -15,7 +15,7 @@ let sigServerB
 let ipfsdServer
 
 module.exports = {
-  bundlesize: { maxSize: '446kB' },
+  bundlesize: { maxSize: '530kB' },
```

Comment:
This is almost a 20% increase in bundle size.  Is there any way this can be made smaller?

Reply:
@jacobheun said, he's gonna improve libp2p-crypto with some ciphers we need for noise. We are currently using bcrypto for those but it's really heavy

Reply:
We will need to get this into `js-ipfs` before `go-ipfs` removes `secio` entirely.
The plan is to work on improving the bundle size with a better solution for libp2p crypto, kind of bring what you need like the rest of libp2p. However, I think that this is something that me or @jacobheun will not be able to get soon.
Some context: https://github.com/libp2p/js-libp2p/issues/586

Reply:
ETAs for secio: https://github.com/libp2p/js-libp2p/issues/615